linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/tests/rhel7_correct_value.pass.sh

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
```

Comment:
We don't use `reboot`, `strategy`, `complexity` and `disruption` keys in test scenarios. See https://complianceascode.readthedocs.io/en/latest/tests/README.html#scenarios-format for description of keys that can be used.